common/reviews/api/rush-lib.api.md

```diff
@@ -375,6 +375,7 @@ export class RushConfiguration {
     readonly repositoryDefaultFullyQualifiedRemoteBranch: string;
     readonly repositoryDefaultRemote: string;
     readonly repositoryUrl: string | undefined;
+    readonly rushConfigurationJson: IRushConfigurationJson;
```

Comment:
Should this be `@internal`? Does anyone need access to the raw JSON?

Reply:
I wanted to use data from the raw JSON here so that I can save some startup time (~80 to 100ms on sp-client) in large monorepos: https://github.com/microsoft/rushstack/pull/2120/files#diff-0d6bb528d5767f5aac6ca9fe00d3e6cd

Reply:
That is an internal reference though, so I agree maybe it should be marked as `@internal`.

Also, it might be worthwhile to investigate why the config file parser is slow.  If we can fix that and speed it up, it would give us a general performance gain across many different operations.

Reply:
I think we spend a lot of time performing exists check and parsing all local package.jsons - which aren't really necessary for some commands such as tab-complete. I will mark this as `@internal`.